app/styles/components/_global.scss

```diff
@@ -175,6 +175,11 @@ textarea {
 /* handle the horizontal alignment ('text-align') properties */
 .text-left {
     text-align: left;
+
+    /* for left-aligned items in the first column of a table, add padding */
+    &.first-col {
```

Comment:
Can we do this with the :first-of-type selector instead of a special class? Just so we don't have to remember to apply it?